scripts/src/public/routes/index.ts

```diff
@@ -63,8 +63,10 @@ function Router(): ExtendedRouter {
 						return (target as any)[name](path, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
 							const token = await authenticate(req);
 							const user = await db.User.findOneById(token.userId);
-							// XXX scopes should be per token and should not consider user data
-							if (scopes.includes(AuthScopes.TOS) && (!user || !user.activated || token.createdDate < user.activatedDate!)) {
+
+							if (!user) {
+								// This error now defines an inconsistent state in the DB where a token exists but not user is found
+								// This should never happen as the token.user_id is a foreign key to the users table
 								throw TOSMissingOrOldToken();
```

Comment:
maybe change the exception to UserWasNotFound or something like that

Reply:
👍